demo/src/actions/app-actions.js

```diff
@@ -41,13 +56,17 @@ export const loadData = (owner, dataArr) => {
       return;
     }
 
-    let resultData = [];
-    let resultMeta = {};
     const isArray = Array.isArray(dataArr);
 
     if (!isArray) {
       dataArr = [dataArr];
     }
+    const context = {
```

Comment:
The word `context` has a lot of baggage. Can we be more specific?

The use is not really clear to me but I assume this is something the worker will gradually populate? (`workerContext`, `workerData`, `workerState` etc).